crates/ln-dlc-node/src/lib.rs

```diff
@@ -48,6 +48,7 @@ pub use ln::EventSender;
 pub use on_chain_wallet::ConfirmationStatus;
 pub use on_chain_wallet::EstimateFeeError;
 pub use on_chain_wallet::TransactionDetails;
+pub use tokio_with_wasm::tokio::spawn;
```

Comment:
I suppose we will need to start using this in `native` too. My understanding this that there will be no downsides for the `coordinator`, given that we won't be compiling for `wasm` there.

Reply:
To avoid problems, we could emit a compiler warning if we use `tokio::spawn` in `ln-dlc-node` and eventually `native`. I think we did something like that in `itchysats`.